notebooks/book1/10/logreg_poly_demo.ipynb

```diff
@@ -1,138 +1,308 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        import probml_utils as pml
</pre><p>This line can be removed because <code>pml</code> is not used anywhere. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/pyprobml/pull/864/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>this is true - we don't need this line. But I guess it's harmless.</p>